packages/frontend/core/src/components/pure/workspace-slider-bar/workspace-card/index.tsx

```diff
@@ -100,49 +125,30 @@ const WorkspaceStatus = ({
     if (!isOnline) {
       return 'Disconnected, please check your network connection';
     }
-    switch (forceSyncStatus.type) {
+    switch (syncEngineStatus) {
       case 'syncing':
         return 'Syncing with AFFiNE Cloud';
       case 'error':
-        return 'Sync failed due to server issues, please try again later.';
+        return 'Sync disconnected due to server issues, reconnecting.';
       default:
-        return 'Sync with AFFiNE Cloud';
+        return 'Synced with AFFiNE Cloud';
     }
-  }, [currentWorkspace.flavour, forceSyncStatus.type, isOnline]);
+  }, [currentWorkspace.flavour, syncEngineStatus, isOnline]);
 
   const CloudWorkspaceSyncStatus = useCallback(() => {
-    if (forceSyncStatus.type === 'syncing') {
+    if (syncEngineStatus === 'syncing') {
       return SyncingWorkspaceStatus();
-    } else if (forceSyncStatus.type === 'error') {
+    } else if (syncEngineStatus === 'error') {
       return UnSyncWorkspaceStatus();
     } else {
       return CloudWorkspaceStatus();
     }
-  }, [forceSyncStatus.type]);
-
-  const handleClick = useCallback(
-    (e: MouseEvent<HTMLDivElement>) => {
-      e.stopPropagation();
-      if (
-        currentWorkspace.flavour === WorkspaceFlavour.LOCAL ||
-        forceSyncStatus.type === 'syncing'
-      ) {
-        return;
-      }
-      startForceSync();
-    },
-    [currentWorkspace.flavour, forceSyncStatus.type, startForceSync]
-  );
+  }, [syncEngineStatus]);
+
   return (
     <div style={{ display: 'flex' }}>
       <Tooltip content={content}>
-        <StyledWorkspaceStatus
-          onMouseEnter={() => {
-            setIsHovered(true);
-          }}
-          onMouseLeave={() => setIsHovered(false)}
-          onClick={handleClick}
-        >
+        <StyledWorkspaceStatus>
```

Comment:
now we don't allow user to manually start sync progress, right?

Reply:
right